nbs/examples/Forecasting_TFT.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<p>Probably plotting here does not really add any value. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Done</p>